java/src/main/kotlin/io/spine/validation/java/DistributingGenerator.kt

```diff
@@ -119,31 +120,32 @@ internal class DistributingGenerator(
         return typeName
     }
 
-    private fun iterableExpression(): Expression {
+    private fun iterableExpression(): Expression<Collection<*>> {
         val fieldAccessor = ctx.fieldOrElement!!
         return if (field.isMap) {
             MethodCall(fieldAccessor, "values")
         } else {
-            fieldAccessor
+            // This generated is applied only to maps or `repeated` fields.
```

Comment:
What do you mean by this sentence? It looks like there are some grammar issues in it.